### PR TITLE
Beacd 56

### PR DESCRIPTION
**Pull request by liampan**

### CODE REVIEW

File: conf/app.routes [hunk 1]
```diff
@@ -166,3 +166,15 @@ POST       /employer/cis/uk/contractor                        controllers.employ
 
 GET        /other/gambling                  controllers.other.gambling.SelectGamblingOrGamingDutyController.onPageLoad()
 POST       /other/gambling                  controllers.other.gambling.SelectGamblingOrGamingDutyController.onSubmit()
+
+GET        /employer/ers/epaye/other-account                       controllers.employer.ers.UseEmployersPAYEController.onPageLoad()
+
+GET        /employer/ers/epaye/not-enrolled                       controllers.employer.ers.AddEmployersPAYEController.onPageLoad()
+
+GET        /employer/ers/register-epaye                       controllers.employer.ers.RegisterEmployersPAYEController.onPageLoad()
```
Comment (darbyo) [inline conf/app.routes]: Space the routes to match the rest of the file

File: conf/messages.en [hunk 2]
```diff
@@ -609,3 +609,32 @@ haveYouRegisteredForVATMOSS.subtext = You’ll have received a VAT MOSS identifi
 haveYouRegisteredForVATMOSS.Yes = Yes - I’ve already registered for VAT MOSS
 haveYouRegisteredForVATMOSS.No = No - I’m not registered for VAT MOSS yet
 haveYouRegisteredForVATMOSS.error.required = Select yes or no
+
+#######################################################
+##  UseEmployersPAYE
+#######################################################
+useEmployersPAYE.title = Use your PAYE for employers account
+useEmployersPAYE.heading = Use your PAYE for employers account
```
Comment (darbyo) [inline conf/messages.en]: should employers have an '

Comment (darbyo) [reply conf/messages.en]: There are a few in this file, just check them all

Comment (liampan) [reply conf/messages.en]: spike raised

**Pull request merged**